chanzuckerberg/single-cell-data-portal
# feat(wmg): Left Side Bar Hover Over Tooltips

## Reason for Change

- #4606

## Changes

- add
- remove
- modify

## Testing steps

1. Hover over new group by, sort cell types, and sort genes help icons
2. Ensure "Please select at least one tissue and gene to use this option." text is shown in tooltip if no tissues/genes are selected
3. Ensure description text and links in tooltip work based on what is in [the ticket](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/4606)
4. Select tissue(s)/gene(s)
5. Hover again over new group by, sort cell types, and sort genes help icons
6. Ensure "Please select at least one tissue and gene to use this option." text is no longer shown in tooltip

rdev: https://left-sidebar-hover-over-frontend.rdev.single-cell.czi.technology/gene-expression

## Notes for Reviewer
